rust/syntect-plugin/src/main.rs

```diff
@@ -249,17 +247,18 @@ impl<'a> Syntect<'a> {
                 //TODO: view should have a `get_edit_builder` fn?
                 let mut builder = EditBuilder::new(buf_size);
                 builder.replace(interval, indent.into());
+
                 let delta = builder.build();
-                return Some(PluginEdit {
-                    rev: rev,
-                    delta: delta,
-                    priority: INDENTATION_PRIORITY,
-                    after_cursor: false,
-                    author: "syntect".to_owned(),
-                })
+                Some(delta)
+            } else {
+                None
+            };
+
+            if let Some(delta) = result {
+                view.edit(delta, INDENTATION_PRIORITY, false,
+                          false, String::from("syntect".to_owned()));
```

Comment:
Didn't notice this earlier, but this is redundant; you can just do `"syntect".into()` or `"syntect".to_string()` _or_ `String::from("syntect")`. Alternatively you could let `view.edit(..)` take a `&str`, and then you can do the conversion internally. Alternatively there's a pattern we use a lot where the function takes a generic `S: AsRef<str>`, see [this nice explanation](https://deterministic.space/elegant-apis-in-rust.html#use-conversion-traits).